### PR TITLE
SDK-705: Provided infrastructure service tests to increase code coverage and quality.

### PR DESCRIPTION
**Pull request by vol4onok**
https://spryker.atlassian.net/browse/SDK-705

### CODE REVIEW

File: tests/Sdk/Unit/Extension/Settings/Initializers/ProjectWorkflowInitializerTest.php [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+use SprykerSdk\Sdk\Infrastructure\Entity\Workflow;
+use SprykerSdk\SdkContracts\Entity\SettingInterface;
+use SprykerSdk\SdkContracts\Entity\WorkflowInterface;
+
+class ProjectWorkflowInitializerTest extends Unit
+{
+    /**
+     * @return void
+     */
+    public function testReturnWhenWorkflowNotFound(): void
```
Comment (pavelmaksimov25) [inline tests/Sdk/Unit/Extension/Settings/Initializers/ProjectWorkflowInitializerTest.php:25]: testReturnWhenWorkflowNotFound => testDoesNothingWhenWorkflowNotFound

File: tests/Sdk/Unit/Extension/Settings/Initializers/ProjectWorkflowInitializerTest.php [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+        $this->expectMethodSaveCall($workflowRepository, false);
+
+        //Act
+        $projectWorkflowInitializer->initialize($setting);
+    }
+
+    /**
+     * @return void
+     */
+    public function testReturnWhenProjectSettingsNotFound(): void
```
Comment (pavelmaksimov25) [inline tests/Sdk/Unit/Extension/Settings/Initializers/ProjectWorkflowInitializerTest.php:43]: testReturnWhenProjectSettingsNotFound => testDoesNothingWhenProjectSettingsNotFound

File: tests/Sdk/Unit/Extension/Settings/Initializers/ProjectWorkflowInitializerTest.php [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+        $this->expectMethodSaveCall($workflowRepository, false);
+
+        //Act
+        $projectWorkflowInitializer->initialize($setting);
+    }
+
+    /**
+     * @return void
+     */
+    public function testCreateAndSaveWhenNoExistingWorkflows(): void
```
Comment (pavelmaksimov25) [inline tests/Sdk/Unit/Extension/Settings/Initializers/ProjectWorkflowInitializerTest.php:61]: it is not clear which case tested from the description. Please, rename

**Review by pavelmaksimov25 — CHANGES_REQUESTED** [review]
Please rename test methods with the names like `testGetValueWithOption`  or `testGetValueException` it is hard to understand what it does from the description.

File: tests/Sdk/Unit/Core/Application/Dto/Abstraction/DtoTest.php [hunk 4]
```diff
@@ -22,6 +22,119 @@
  */
 class DtoTest extends Unit
 {
+    /**
+     * @return void
+     */
+    public function testDtoCreate(): void
+    {
+        // Arrange
+        $dto = new class extends Dto {
```
[the remark this answers is not in the document]
Comment (vol4onok) [reply tests/Sdk/Unit/Core/Application/Dto/Abstraction/DtoTest.php:31]: we don't use data list here. it's not need for this test

File: tests/Sdk/Unit/Core/Application/Dto/Abstraction/DtoTest.php [hunk 5]
```diff
@@ -22,6 +22,119 @@
  */
 class DtoTest extends Unit
 {
+    /**
```
[the remark this answers is not in the document]
Comment (vol4onok) [reply tests/Sdk/Unit/Core/Application/Dto/Abstraction/DtoTest.php:25]: +

File: tests/Sdk/Unit/Extension/Settings/Initializers/CreateDirectoryInitializerTest.php [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+        $setting = $this->createSettingInterfaceMock($pathDirPath);
+
+        //Act
+        $createDirectoryInitializer->initialize($setting);
+
+        //Assert
+        $this->assertTrue($vfsStream->hasChild('settingDir'));
+    }
+
+    /**
```
[the remark this answers is not in the document]
Comment (vol4onok) [reply tests/Sdk/Unit/Extension/Settings/Initializers/CreateDirectoryInitializerTest.php:35]: +

File: tests/Sdk/Unit/Extension/Settings/Initializers/ProjectUuidInitializerTest.php [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+use SprykerSdk\Sdk\Core\Appplication\Dependency\ProjectSettingRepositoryInterface;
+use SprykerSdk\Sdk\Extension\Settings\Initializers\ProjectUuidInitializer;
+use SprykerSdk\SdkContracts\Entity\SettingInterface;
+
+class ProjectUuidInitializerTest extends Unit
+{
+    /**
+     * @return void
+     */
+    public function testSetUuidWhenSettingNotFound(): void
```
[the remark this answers is not in the document]
Comment (vol4onok) [reply tests/Sdk/Unit/Extension/Settings/Initializers/ProjectUuidInitializerTest.php:20]: renamed to testInitializeWhenSettingNotFound and for other.

**Review by sergeyspryker — APPROVED** [review]

**Pull request merged**